tyk-pro/values.yaml

```diff
@@ -213,6 +213,7 @@ dash:
   service:
     type: NodePort
     port: 3000
+    nodePort: 30001
```

Comment:
I think having this set to a particular port would affect the service that are not running in a KinD cluster. Maybe move this whole section into an example directory and link that in the `README.md`